js/views/navigation.js

```diff
@@ -46,6 +51,11 @@ define(function(require) {
 				this.$el.hide();
 				$('#app-navigation-toggle').css('background-image', 'none');
 			}
+		},
+		_adjustButtonWidth: function() {
+			var newMailButtonContainer = $('#mail-new-message-fixed');
+			var innerWidth = $('#app-navigation-accounts').width();
+			newMailButtonContainer.attr('style', 'width: ' + innerWidth + 'px !important');
```

Comment:
Navigation entries have 100% width. As the button has a fixed position, this would make it as wide as the browser window. Hence, the width is enforced on that specific element. I tried to remove it, but I couldn't get it to an acceptable state. If you know a fix for that, let me know ;-)